### PR TITLE
Update README.md

### LINKED ISSUE #1: Add to F-Droid
...as I don't see any closed source deps this should be fine, right?

### LINKED ISSUE #2: master
open : #1 Add to F-Droid

### PR DESCRIPTION
**Pull request by Poussinou**
Hi,

This small PR adds two buttons to get your app on F-Droid and Google Play 👍 It also adds some explanation, taken from F-Droid/Google Play pages.

close #1 
close #2

### CODE REVIEW

**Review by vodafone001 — APPROVED** [review]

**Comment by Krit2521** [comment]
Thank

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-# dontkillmy-app
-Tool for benchmarking nonstandard background processing limitations that are done by OEMs on different Android flavours.
```
Comment (QUANGHIEN190589) [inline README.md:2]:
```

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
-# dontkillmy-app
```
Comment (QUANGHIEN190589) [inline README.md:1]: -

**Review by QUANGHIEN190589 — APPROVED** [review]

**Comment by yashpalgoyal1304** [comment]
hi, any updates on why not merged? or do u not wanna have full description here? if so, then only the buttons for fdroid and gp can be added too...

**Review by KibzGithub7407 — APPROVED** [review]
Do not kill myApps

**Comment by yashpalgoyal1304** [comment]
2 approverals, more than 20 months, what more are we waiting for (:

**Review by Artaud — APPROVED** [review]

**Comment by Artaud** [comment]
Sorry about the wait, this is awesome PR, thank you very much!

**Pull request merged**